### PR TITLE
55 fix tests upgrade dependencies

### LINKED ISSUE #55: Fix tests & update dependencies

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
Resolves #55

### CODE REVIEW

File: src/spec/helper/helper.coffee [hunk 1]
```diff
@@ -41,3 +41,30 @@ exports.createReader = (fileType, delimiter = ',', encoding = 'utf-8' ) ->
     csvDelimiter: delimiter,
     encoding: encoding,
     importFormat: fileType,
+
+exports.unpublishAllProducts = (client) ->
```
Comment (wizzy25) [inline src/spec/helper/helper.coffee]: I don't think you need to export this function as it's not used anywhere else

File: src/spec/integration/product-type-importer.spec.coffee [hunk 2]
```diff
@@ -75,14 +76,16 @@ describe 'ProductTypeImporter', ->
...
 
-    sphereClient.productTypes
-    .perPage(50)
-    .process (res) ->
-      console.log "Deleting old product types", res.body.results.length
-      Promise.map res.body.results, (productType) ->
-        sphereClient.productTypes.byId(productType.id)
-        .delete(productType.version)
-      , concurrency: 10
+    Helper.deleteAllProducts sphereClient
```
Comment (wizzy25) [inline src/spec/integration/product-type-importer.spec.coffee]: You could also move this function to the helper file so you export only this one and it calls the others implicitly

File: src/spec/helper/helper.coffee (hunk 1 above)
Comment (junajan) [inline src/spec/helper/helper.coffee]: Hello @wizzy25 - updated here fa6e1da

File: src/spec/integration/product-type-importer.spec.coffee (hunk 2 above)
Comment (junajan) [inline src/spec/integration/product-type-importer.spec.coffee]: Hello @wizzy25 - updated here fa6e1da

**Pull request merged**